### PR TITLE
bpo-32627: Fix compile error when `_uuid` headers conflicting included

### PR DESCRIPTION
**Pull request by zihengCat**
A compile error occurred when operating system both have `uuid.h` and `uuid/uuid.h`.

- Versions: *Python 3.8*, *Python 3.7*

https://bugs.python.org/issue32627

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Unfortunately we couldn't find an account corresponding to your GitHub username on [bugs.python.org](https://bugs.python.org/) (b.p.o) to verify you have signed the CLA (this might be simply due to a missing "GitHub Name" entry in your b.p.o account settings). This is necessary for legal reasons before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.


You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for your contribution, we look forward to reviewing it!

**Review by eamanu — COMMENTED** [review]
lgtm. We need wait for a core dev review it. 

Please, sign CLA.

**Comment by zihengCat** [comment]
@eamanu 
I couldn't compile Python3.7 successfully in CentOS 7 Linux because of this bug, seems that it's also occurred in Fedora as well as NetBSD.

**Review by ned-deily — APPROVED** [review]

**Pull request merged**
